SUIT FOTA updates section (#19557)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

> :orphan:

Reviewer: I wonder if we could e.g. link the page under https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_dfu.html or add the content as an example on https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_smp.html (to avoid orphan page)

Author: This is up to the Charon and the Charon documentation teams to decide. My intention is to create linkable steps for SUIT-based applications, not a full-blown documentation page.

Author: @tomchy recommended to set up this page as an orphan page and that's why I did it like that (probably due to the upcoming documentation restructure)

Reviewer 2: @kapi-no are these steps only supposed to be "quoted" from other doc pages? If this is the only scope of this PR, this should be done through a txt document located in doc\nrf\includes

.. _ug_nrf54h20_suit_fota_update:

FOTA updates on nRF54H Series devices
#####################################

> 

Reviewer 3: .. contents::
   :local:
   :depth: 2

Please add the above.

|fota_upgrades_def|
You can also use FOTA updates to replace the application.

> FOTA over Bluetooth Low Energy
> ******************************

Reviewer 3: ```suggestion
FOTA over Bluetooth® Low Energy
*******************************
```

Author: this needs to be aligned in our active documentation (see nRF52/53/54L guides)

FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.

> The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.

Reviewer: ```suggestion
The application acts as a GATT server and allows the connected GATT Client to perform a firmware update.
```

> To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).

Reviewer 3: ```suggestion
To use FOTA over Bluetooth LE, samples must support the Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
```

Testing steps
=============

> To perform a FOTA update, complete the following steps:

Reviewer: Or are the "more advanced" scenarios already supported by the mobile applications "under the hood".
@tomchy could you advise on that, please?

> To perform a FOTA update, complete the following steps:

Author: This can be changed or improved once this documentation page is part of the public documentation set (orphan). This sentence is not visible to any user right now.

> To perform a FOTA update, complete the following steps:

Reviewer 4: The "more advanced" scenarios are supported if you use the `.zip` file instead of the raw `.suit` envelope.

.. ug_nrf54h20_suit_fota_updates_over_ble_nrfcdm_common_dfu_steps_start

1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
   You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
   Alternatively, you can use the generated :file:`dfu_suit.zip` package in the :file:`<build_dir>/zephyr` directory.

> #. Download the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package to your device.

Reviewer: `to your mobile device running the DFU mobile app` (to be more precise)

Reviewer: Alternatively, we could skip this step as the subsequent step already states that 
```
a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
```

Author: Correct point 2 but kept the statement from point a

   .. note::
      `nRF Connect for Desktop`_ does not currently support the FOTA process.

#. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.

   a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
   #. In the mobile app, scan and select the device to update.
   #. Switch to the :guilabel:`Image` tab.
   #. In the **Firmware Upgrade** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package.
   #. Tap the :guilabel:`START` button.
   #. Wait for the DFU to finish and then verify that the application works properly.

.. note::
   Support for SUIT updates is available starting from the following versions of the `nRF Connect Device Manager`_ mobile app:

   * Version ``2.0`` on Android.

>    * Version ``1.7`` on iOS.

Reviewer 4: Due to the recent modification in FW confirmation, the current set of version is:
```
* `nRF Device Manager mobile app for Android`_
  (The minimum required version is v2.2.1.)

* `nRF Device Manager mobile app for iOS`_
  (The minimum required version is v1.8.0.)
  ```

Author: fixed
